pkg/stream/client.go

```diff
@@ -147,6 +148,15 @@ func (c *client) OnEvent(event types.ConnectionEvent) {
 
 		if c.ConnectedFlag {
 			reason = types.StreamConnectionTermination
+
+			if c.Protocol == protocol.HTTP1 {
```

Comment:
http is not visible to the stream layer

Reply:
Yeah, In order to be compatible with exception scenarios in the HTTP  and not affect the processing logical of other protocols. so hardcode here.   
TODO
compatible with all protocols about `event` and `reason`.